dg/media/app_dashboards/js/training_dashboard.js

```diff
@@ -523,59 +712,72 @@ function plot_statewise_data(state_list, mediator_list) {
-        plot_dual_axis_chart($("#state_mediator_data"), x_axis, state_scores_dict, "Average Scores per Mediator", "Percent Answered Correctly", "", "%");
-        plot_multiple_axis_chart($("#state_training_data"), x_axis, state_trainings_mediators_dict, "Trainings", "Mediators", "% Mediators above 70%", "", "", "%");
+        plot_dual_axis_chart($("#state_mediator_data"), x_axis, state_scores_dict, "Average Scores per Mediator", "", "", "%","");
+        plot_multiple_axis_chart($("#state_training_data"), x_axis, state_trainings_mediators_dict, "No. of Mediators", "", "", "", "", "%", state_trainings_dict, "", "Labels represent : No. of total trainings","");
     }
 }
 
-/* plot highcharts data */
+function plot_monthwise_data(series_name, series_data_list) {
 
-function plot_piechart(container_obj, _data, arg) {
-    var chart = {
-        plotBackgroundColor: null,
-        plotBorderWidth: null,
-        plotShadow: false
-    };
+        $('#month_training_data').highcharts({
+        chart: {
+            type: 'column'
+        },
+        credits:{enabled :false},
+        title: {
+            text: 'No. Of Trainings Per Month'
+        },
+        xAxis: {
+            categories: [
+                'Jan',
+                'Feb',
+                'Mar',
+                'Apr',
+                'May',
+                'Jun',
+                'Jul',
+                'Aug',
+                'Sep',
+                'Oct',
+                'Nov',
+                'Dec'
+            ],
+            crosshair: true
+        },
+        yAxis: {
+            min: 0,
+            title: {
+                text: 'No. of trainings'
+            }
+        },
+        tooltip: {
+            shared: true,
+        },
 
-    var tooltip = {
-        pointFormat: '{series.name}: <b>{point.percentage:.1f}%</b>'
-    };
-    var plotOptions = {
-        pie: {
-            allowPointSelect: true,
-            cursor: 'pointer',
-            dataLabels: {
+        plotOptions: {
+            column: {
+                pointPadding: 0.2,
+                borderWidth: 0,
+                dataLabels: {
                 enabled: true,
-                format: '<b>{point.name}%</b>: {point.percentage:.1f} %',
-                style: {
-                    color: (Highcharts.theme && Highcharts.theme.contrastTextColor) || 'black'
-                }
+                format:'{point.y}'
             }
-        }
-    };
-
-    series = [{
-        type: 'pie',
-        name: arg,
-        data: _data
-    }];
-
-    var json = {};
-    json.chart = chart;
-    json.title = null;
-    json.tooltip = tooltip;
-    json.series = series;
-    json.plotOptions = plotOptions;
-    container_obj.highcharts(json);
+            }
+        },
+        series: [{
+            name: series_name,
+            data: series_data_list
+       
+            }]
+    });
 }
 
+/* plot highcharts data */
+
 function plot_multiline_chart(container_obj, x_axis, dict, y_axis_text) {
```

Comment:
This function is not being used anywhere in the code. Can be removed.
Also plot_single_axis_chart and plot_stacked_chart are not being used anywhere.